### PR TITLE
#548 - Tests for GetStorageSlice

### PR DESCRIPTION
**Pull request by olegmoz**
Part of #548 
Added tests for `GetStorageSlice`

### CODE REVIEW

File: src/test/java/com/artipie/api/artifactory/GetStorageSliceTest.java [hunk 1]
```diff
@@ -0,0 +1,132 @@
...
+     */
+    @TempDir
+    Path tmp;
+
+    @ParameterizedTest
+    @CsvSource({
+        "flat,my-lib",
+        "org,my-company/my-lib"
+    })
+    void shouldReturnExpectedData(final String layout, final String repo) {
```
Comment (olenagerasimova) [inline src/test/java/com/artipie/api/artifactory/GetStorageSliceTest.java:77]: @olegmoz `layout` is not used, maybe we can remove it and use `@ValueSource` here

**Review by olenagerasimova — CHANGES_REQUESTED** [review]
@olegmoz just one comment, please, take a look

File: src/test/java/com/artipie/api/artifactory/GetStorageSliceTest.java (hunk 1 above)
Comment (olegmoz) [reply src/test/java/com/artipie/api/artifactory/GetStorageSliceTest.java:77]: @olenagerasimova it is not used now, but it should be used later. `GetStorageSlice` accepts `Storage` as parameter now, but it should be changed to accept `Settings`, extract repo name from request path using `Settings.layout` and then take `Storage` by repo name. So I started off with wrtiting proper test and will change `GetStorageSlice` to do all that later

**Comment by olegmoz** [comment]
@olenagerasimova answered your question, please review

**Review by olenagerasimova — APPROVED** [review]
@olegmoz thanks for the explanation, I see now

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#587` is not assigned, can't get performer

**Pull request merged**